mono_repo/lib/src/commands/github/github_yaml.dart

```diff
@@ -222,28 +227,95 @@ Map<String, dynamic> _createDartSetup(String sdk) {
   return map;
 }
 
-Map<String, dynamic> _githubJobYaml(String jobName, String jobOs,
-        String dartVersion, Map<String, Map<String, dynamic>> runCommands) =>
+Map<String, dynamic> _githubJobYaml(
```

Comment:
Can you add a doc comment that explains the arguments? In particular I'm not sure without a doc what `packages` argument is.